scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+    users.
+- group: frontend
+  name: antenna_selection
+  type: enum
+  units:
+  default value: Auto
+  enumerated possible values: Auto,Patch,External
+  Description: Determines which antenna to use.
+  Notes: This setting selects the antenna input that should be used by the Piksi.  When
+    set to auto, if the unit senses an external antenna attached, it will use the
+    external antenna.  If no external antenna is attached, it will use the integrated
+    patch antenna.  This setting can override the automatic antenna selection and
+    force the external or patch antenna to be used.
+- group: iar
+  name: phase_var
+  type: double
+  units: cycles^2
+  default value: '0.0144'
+  enumerated possible values:
+  Description: Determines the measured carrier phase variance for use in the integer
+    ambiguity resolution test loop
+  Notes: This setting adjusts variance estimates in the integer ambiguity resolution
+    (IAR) subroutine.  Increasing this value can reduce the occurrence of false carrier
+    phase locks but can also increase the time required to achieve an IAR fixed solution.  This
+    setting should not be adjusted by end users.
+- group: iar
+  name: code_var
+  type: double
+  units: meters^2
+  default value: '40000'
+  enumerated possible values:
+  Description: Determines the pseudocode variance for the integer ambiguity resolution
+    subroutine
+  Notes: This setting adjusts variance estimates in the integer ambiguity resolution
+    (IAR) subroutine.  Increasing this value can reduce the occurrence of false carrier
+    phase locks but can also increase the time required to achieve an IAR fixed solution.  This
+    setting should not be adjusted by end users.
+- group: sbp
+  name: obs_msg_max_size
+  type: integer
+  units: bytes
+  default value: '104'
+  enumerated possible values:
+  Description: Determines the maximum message length for raw observation sbp messages.
+  Notes: This parameter is useful for tuning observation messages for compatibility
+    with radio modems.  Some serial modems will internally split serial packets for
+    their protocol and this parameter allows the size of the message to be reduced
+    as to prevent the modem from sending multiple packets. If the parameter exceeds
+    255 bytes (the maximum size of an SBP message), the Piksi firmware will ignore
+    the parameter and use 255 bytes.  If the parameter is set smaller than the size
+    of one observation, the Piksi firmware will ignore the parameter and use the size
+    of one observation as the maximum message size.
+- group: simulator
+  name: enabled
+  type: boolean
+  units:
+  default value: 'false'
+  enumerated possible values: true,false
+  Description: Toggles the swift navigation internal simulator on and off
+  Notes: The swift navigation simulator will provide simulated outputs of a stationary
+    base station and the Local Piksi moving in a circle around the base station.
```

Comment:
Maybe make a note that the simulator is intended to aid in systems integration and doesn't attempt to faithfully simulate every aspect of the device operation.